serverless.yml

```diff
@@ -116,8 +129,9 @@ stepFunctions:
             Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-capture-field-visit-transform-${self:provider.stage}-transform
             Parameters:
               id.$: $.id
+              partitionNumber.$: $.partitionNumber
               type.$: $.type
```

Comment:
You'll need to change this too because is type is now found in `$.tsTypeRouterResult.type`.